pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -92,7 +93,10 @@ func ConvReason(status libvirt.DomainState, reason int) api.StateChangeReason {
 }
 
 func SetDomainSpecStr(virConn cli.Connection, vmi *v1.VirtualMachineInstance, wantedSpec string) (cli.VirDomain, error) {
-	log.Log.Object(vmi).V(3).With("xml", wantedSpec).Info("Domain XML generated.")
+
+	fmt.Println("Domain XML generated.")
+	fmt.Println(convertToLibvirtXML(wantedSpec))
```

Comment:
Don't we want to use log ?

Reply:
+1 would that mean that we just drop a xml in the middle of the log stream?

Reply:
@qinqon @rmohr the logger we use is based on JSON logger and it is set with escapeHTMLLtrue which means the >< & characters will be escaped. I don't see an option to override this value when calliing to log.Log(..) @rmohr the log stream looks coherent with this print, so per the validations I made on local setup.

Reply:
@enp0s3 can you paste a chunk of how log looks like ?

Reply:
@qinqon Sure. [link](https://raw.githubusercontent.com/enp0s3/tools/master/xml_log.txt)

Reply:
What is the issue with the escapes?

Reply:
@rmohr It's not human readable, for example, it will look like this:
```
"\u003cdomain type=\"kvm\"\u003e\n\t\u003cname\u003edefault_vmi-ephemeral\u003c/name\u003e\n\t\u003cmetadata\u003e\u003ckubevirt xmlns=\"http://kubevirt.io\"\u003e\u003cuid\u003e1a978d61-a6a0-4f37-a460-ec3369c813d0\u003c/uid\u003e\u003cgraceperiod\u003e\u003cdeletionGracePeriodSeconds\u003e0\u003c/deletionGracePeriodSeconds\u003e\u003c/graceperiod\u003e\u003c/kubevirt\u003e\u003c/metadata\u003e\n\t\u003cmemory unit=\"b\"\u003e64000000\u003c/memory\u003e\n\t\u003cvcpu placement=\"static\"\u003e1\u003c/vcpu\u003e\n\t\u003ciothreads\u003e1\u003c/iothreads\u003e\n\t\u003csysinfo type=\"smbios\"\u003e\n\t\t\u003cbios\u003e\u003c/bios\u003e\n\t\t\u003csystem\u003e\n\t\t\t\u003centry name=\"uuid\"\u003e44c70f30-2bb3-4b54-8e9d-22046d72df3f\u003c/entry\u003e\n\t\t\t\u003centry name=\"manufacturer\"\u003eKubeVirt\u003c/entry\u003e\n\t\t\t\u003centry name=\"family\"\u003eKubeVirt\u003c/entry\u003e\n\t\t\t\u003centry name=\"product\"\u003eNone\u003c/entry\u003e\n\t\t\t\u003centry name=\"sku\"\u003e\u003c/entry\u003e\n\t\t\t\u003centry name=\"version\"\u003e\u003c/entry\u003e\n\t\t\u003c/system\u003e\n\t\t\u003cbaseBoard\u003e\u003c/baseBoard\u003e\n\t\t\u003cchassis\u003e\u003c/chassis\u003e\n\t\u003c/sysinfo\u003e\n\t\u003cos\u003e\n\t\t\u003ctype arch=\"x86_64\" machine=\"q35\"\u003ehvm\u003c/type\u003e\n\t\t\u003csmbios mode=\"sysinfo\"\u003e\u003c/smbios\u003e\n\t\u003c/os\u003e\n\t\u003cfeatures\u003e\n\t\t\u003cacpi\u003e\u003c/acpi\u003e\n\t\u003c/features\u003e\n\t\u003ccpu mode=\"host-model\"\u003e\n\t\t\u003ctopology sockets=\"1\" cores=\"1\" threads=\"1\"\u003e\u003c/topology\u003e\n\t\u003c/cpu\u003e\n\t\u003cdevices\u003e\n\t\t\u003cdisk type=\"file\" device=\"disk\"\u003e\n\t\t\t\u003cdriver name=\"qemu\" type=\"qcow2\" cache=\"none\"\u003e\u003c/driver\u003e\n\t\t\t\u003csource file=\"/var/run/kubevirt-ephemeral-disks/disk-data/containerdisk/disk.qcow2\"\u003e\u003c/source\u003e\n\t\t\t\u003cbackingStore type=\"file\"\u003e\n\t\t\t\t\u003cformat type=\"qcow2\"\u003e\u003c/format\u003e\n\t\t\t\t\u003csource file=\"/var/run/kubevirt/container-disks/disk_0.img\"\u003e\u003c/source\u003e\n\t\t\t\u003c/backingStore\u003e\n\t\t\t\u003ctarget dev=\"vda\" bus=\"virtio\"\u003e\u003c/target\u003e\n\t\t\t\u003calias name=\"ua-containerdisk\"\u003e\u003c/alias\u003e\n\t\t\u003c/disk\u003e\n\t\t\u003ccontroller type=\"usb\" index=\"0\" model=\"none\"\u003e\u003c/controller\u003e\n\t\t\u003ccontroller type=\"virtio-serial\" index=\"0\"\u003e\u003c/controller\u003e\n\t\t\u003cinterface type=\"bridge\"\u003e\n\t\t\t\u003cmac address=\"0e:7d:82:83:7f:3a\"\u003e\u003c/mac\u003e\n\t\t\t\u003csource bridge=\"k6t-eth0\"\u003e\u003c/source\u003e\n\t\t\t\u003cmodel type=\"virtio\"\u003e\u003c/model\u003e\n\t\t\t\u003cmtu size=\"1440\"\u003e\u003c/mtu\u003e\n\t\t\t\u003calias name=\"ua-default\"\u003e\u003c/alias\u003e\n\t\t\u003c/interface\u003e\n\t\t\u003cserial type=\"unix\"\u003e\n\t\t\t\u003csource mode=\"bind\" path=\"/var/run/kubevirt-private/1a978d61-a6a0-4f37-a460-ec3369c813d0/virt-serial0\"\u003e\u003c/source\u003e\n\t\t\t\u003ctarget port=\"0\"\u003e\u003c/target\u003e\n\t\t\u003c/serial\u003e\n\t\t\u003cconsole type=\"pty\"\u003e\n\t\t\t\u003ctarget type=\"serial\" port=\"0\"\u003e\u003c/target\u003e\n\t\t\u003c/console\u003e\n\t\t\u003cchannel type=\"unix\"\u003e\n\t\t\t\u003ctarget type=\"virtio\" name=\"org.qemu.guest_agent.0\"\u003e\u003c/target\u003e\n\t\t\u003c/channel\u003e\n\t\t\u003cgraphics type=\"vnc\"\u003e\n\t\t\t\u003clisten type=\"socket\" socket=\"/var/run/kubevirt-private/1a978d61-a6a0-4f37-a460-ec3369c813d0/virt-vnc\"\u003e\u003c/listen\u003e\n\t\t\u003c/graphics\u003e\n\t\t\u003cvideo\u003e\n\t\t\t\u003cmodel type=\"vga\" heads=\"1\" vram=\"16384\"\u003e\u003c/model\u003e\n\t\t\u003c/video\u003e\n\t\t\u003cmemballoon model=\"virtio\"\u003e\n\t\t\t\u003cstats period=\"10\"\u003e\u003c/stats\u003e\n\t\t\u003c/memballoon\u003e\n\t\u003c/devices\u003e\n\u003c/domain\u003e\n","pos":"libvirt_helper.go:100","timestamp":"2020-08-05T15:49:46.732033Z"}
```